### PR TITLE
Fixes clicking on prereqs

### PR DESCRIPTION
**Pull request by NEUDitao**
addresses https://github.com/ryanhugh/searchneu/issues/139

### CODE REVIEW

File: frontend/components/panels/BaseClassPanel.js [hunk 1]
```diff
@@ -209,14 +209,13 @@ class BaseClassPanel extends React.Component {
 
           // When adding support for right click-> open in new tab, we might also be able to fix the jsx-a11y/anchor-is-valid errors.
           // They are disabled for now.
-          const hash = `/${this.props.aClass.termId}/${childBranch.subject} ${childBranch.classId}`;
+          const hash = `/${this.props.aClass.termId}/${childBranch.subject}${childBranch.classId}`;
```
Comment (gamburgm) [inline frontend/components/panels/BaseClassPanel.js:212]: You need another slash b/w subject and classId, right?

Comment (gamburgm) [reply frontend/components/panels/BaseClassPanel.js:212]: Would be good to write a test (if possible) rendering some example data and making sure it renders with the right data here.

Also, can't we use `Keys.getClassHash` here?

File: frontend/components/ResultsLoader.tsx [hunk 2]
```diff
@@ -39,14 +40,16 @@ function ResultsLoader({ results, loadMore }: ResultsLoaderProps) {
...
 // If the Panels are updated to function components, we can memoize them instead and remove this
 const ResultItemMemoized = React.memo(({ result }:{result:SearchItem}) => {
+  const history = useHistory();
+
   if (result.type === 'class') {
     const aClass = Class.create(result.class);
     aClass.loadSectionsFromServerList(result.sections);
     if (macros.isMobile) {
-      return <MobileClassPanel aClass={ aClass } />;
+      return <MobileClassPanel aClass={ aClass } history={ history } />;
```
Comment (gamburgm) [inline frontend/components/ResultsLoader.tsx:49]: Why is history necessary for fixing this guy? I thought we had the functionality working for back button leading to previous search?

Comment (NEUDitao) [reply frontend/components/ResultsLoader.tsx:49]: history is the hook needed for things to redirect on click

File: frontend/components/panels/BaseClassPanel.js (hunk 1 above)
Comment (NEUDitao) [reply frontend/components/panels/BaseClassPanel.js:212]: No on having the slash between subject asnd classId, adn no on `Keys.getClassHash`. we're essentially doing a search here, not going to a page of any sort

Comment (gamburgm) [reply frontend/components/panels/BaseClassPanel.js:212]: oh my b

**Review by gamburgm — APPROVED** [review]

**Review by dajinchu — APPROVED** [review]

**Pull request merged**